absearch/server.py

```diff
@@ -134,20 +137,29 @@ def handle_500_error(code):
 
 @app.route(PATH)
 def add_user_to_cohort(**kw):
-
-    with app._statsd.timer('add_user_to_cohort'):
+    if app._statsd:
+        with app._statsd.timer('add_user_to_cohort'):
+            res = app.settings.get(**kw)
+    else:
         res = app.settings.get(**kw)
+
+    if app._statsd and app._config['statsd']['count_cohorts_enabled']:
         cohort = res.get('cohort', 'default')
         cohort = '.'.join(['cohorts', kw['locale'],
```

Comment:
Could you also rename the key from ``cohorts.locale.territory.cohort`` to to ``cohorts.locale_territory_cohort`` ?